apps/web/src/components/forms/password.tsx

```diff
@@ -79,6 +83,14 @@ export const PasswordForm = ({ className }: PasswordFormProps) => {
     }
   };
 
+  const onShowPassword = () => {
+    setShowPassword(!showPassword);
```

Comment:
We typically prefer the `setX(x => !x)` pattern for these since if they’re added into effect that can start to loop infinitely 😄

Reply:
I will update that :)